main.tf

```
@@ -203,8 +203,8 @@ resource "aws_lb_listener" "https" {
   tags            = merge(module.this.tags, var.listener_additional_tags)
 
   default_action {
-    target_group_arn = var.listener_https_fixed_response != null ? null : one(aws_lb_target_group.default[*].arn)
-    type             = var.listener_https_fixed_response != null ? "fixed-response" : "forward"
+    target_group_arn = var.listener_https_fixed_response != null ? null : var.listener_https_redirect != null ? null : one(aws_lb_target_group.default[*].arn)
```

```suggestion
    target_group_arn = var.listener_https_fixed_response != null || var.listener_https_redirect != null ? null : one(aws_lb_target_group.default[*].arn)
```
It could be shortened a bit like this, right? I think it's easier to read.